stores/sql/main.go

```diff
@@ -56,6 +57,24 @@ func AbortMultipartUpload(ctx context.Context, tx sql.Tx, bucket, key string, up
 	return errors.New("failed to delete multipart upload for unknown reason")
 }
 
+func Accounts(ctx context.Context, tx sql.Tx) ([]api.Account, error) {
+	rows, err := tx.Query(ctx, "SELECT account_id, clean_shutdown, host, balance, drift, requires_sync FROM ephemeral_accounts")
+	if err != nil {
+		return nil, fmt.Errorf("failed to fetch accounts: %w", err)
+	}
+	defer rows.Close()
+
+	var accounts []api.Account
+	for rows.Next() {
+		a := api.Account{Balance: new(big.Int), Drift: new(big.Int)} // init big.Int
```

Comment:
what's does the comment mean?

Reply:
It means we init the big.Int fields since they are otherwise `nil`.